user/user-tool-prefs/tool/src/java/org/sakaiproject/user/tool/UserPrefsTool.java

```diff
@@ -343,6 +349,24 @@ public String getSelectedEditorType()
 		return m_editorType;
 	}
 
+	/**
+	 * @return Returns the user's selected Sakai theme
+	 */
+	public String getSelectedTheme()
+	{
+		if (m_theme != null) return m_theme;
+
+		Preferences prefs = (PreferencesEdit) preferencesService.getPreferences(getUserId());
+		String userTheme = StringUtils.defaultIfEmpty(prefs.getProperties(org.sakaiproject.user.api.PreferencesService.USER_SELECTED_UI_THEME_PREFS).getProperty("theme"), "sakaiUserTheme-notSet");
+
+		if (hasValue(userTheme))
```

Comment:
You should use curlies here, chief. Even if you have one line it still makes the code more robust, in case somebody else just adds another indented line and expects that magical Python scoping will just happen.